app/helpers/application_helper/button/vm_instance_scan.rb

```diff
@@ -3,14 +3,12 @@ class ApplicationHelper::Button::VmInstanceScan < ApplicationHelper::Button::Bas
 
   def skip?
     return false if @record.kind_of?(OrchestrationStack) && @display == "instances"
-    !(@record.is_available?(:smartstate_analysis) ||
-      @record.is_available_now_error_message(:smartstate_analysis)) ||
-    !@record.has_proxy?
+    @record.archived? || !@record.has_proxy?
```

Comment:
why isnt this using `supports_smartstate_analysis?` and why is there the `archived?` check?